Makefile

```diff
@@ -0,0 +1,34 @@
+# author: Anh Le
+# date: Mar 12 2020
+# makefile for Group4 STAT 547
+
+# download data
+aq.csv : Scripts/load_data.R
+	load_data.R Rscript Scripts/load_data.R --url="https://raw.githubusercontent.com/STAT547-UBC-2019-20/data_sets/master/airquality.csv" --outfilename="aq.csv"
+
+# clean data
+clean_aq.csv : Scripts/clean_data.R data/clean_data.csv
+	Rscript Scripts/clean_data.R --data_dir="Data" --infilename="aq.csv" --outfilename="clean_aq.csv"
+
+# EDA
+Images/correlation.png Images/pollutantsvstime.png Images/weathervstime.png Images/tempvsbenzene.png : EDA.R Data/clean_aq.csv
+	Rscript Scripts/EDA.R --data_dir="Data" --datafilename="clean_aq.csv"
+	
+# LinearRegression
+Images/CoefPlot_Group4.png CO.rds, Tin_oxide.rds, Hydro_carbons.rds, Benzene.rds, Titania.rds, NOx.rds, Tungsten_oxide_NOx.rds, NO2.rds, Tungsten_oxide_NO2.rds, 
```

Comment:
`Images/CoefPlot_Group4.png CO.rds, Data/Tin_oxide.rds, Data/Hydro_carbons.rds, Data/Benzene.rds, Data/Titania.rds, Data/NOx.rds, Data/Tungsten_oxide_NOx.rds, Data/NO2.rds, Data/Tungsten_oxide_NO2.rds, Data/Indium_oxide.rds : LinearRegression.R Data/clean_aq.csv`

I would change that so that the `.rds` files get saved in the data folder... otherwise they're all just in the main folder

Reply:
True I'll fix it